pkg/check/datastore.go

```diff
@@ -111,9 +110,17 @@ func CheckStorageClasses(ctx *CheckContext) error {
 		for k, v := range sc.Parameters {
 			switch strings.ToLower(k) {
 			case dsParameter:
-				if err := checkDataStore(ctx, v, ctx.VMConfig.Workspace.Datacenter, dsTypes); err != nil {
-					klog.V(2).Infof("CheckStorageClasses: %s: %s", sc.Name, err)
-					errs = append(errs, fmt.Errorf("StorageClass %s: %s", sc.Name, err))
+				// Its like we can check multiple datacenters, but we assumed here only 1 value in the field.
+				if ctx.VMConfig.LegacyConfig != nil {
+					// Legacy will only have 1 vCenter.  Looping just to get first and only vCenter.
+					for _, vCenter := range ctx.VCenters {
```

Comment:
ctx.VCenters is a map.  I can change this to use either key instead of _, or I could do ctx.VMConfig.LegacyConfig.Workspace.VCenterIP for the expected key value.  I am fine doing the later.